sdk/schemaregistry/schema-registry-json/src/cache.ts

```diff
@@ -66,3 +54,10 @@ export function getSchemaObject(schema: string): SchemaObject {
     `Parsing Json schema failed:\n\n\t${schema}\n\nSee 'cause' for more details.`
   );
 }
+
+export interface SchemaObject {
+  id?: string;
```

Comment:
curious that there is both `id` and `$id`

Reply:
I'll remove the `id`. The keyword in JSON Schema is `$id`.